steps/gitignore.ts

```diff
@@ -1,10 +1,11 @@
 import type { Init } from "../init.ts";
 
-export default function () {
+export default function() {
   return ({ files }: Init) => {
     const paths = [
-      "_site",
-      "_cache",
+      "# Ignore IDEA settings \n .vscode \n .idea \n",
```

Comment:
This is too opinionated. Ignoring `.vscode` is not always a good idea, specially if you want to share the settings with other people.
I prefer to add only Lume specific stuff.